redash/models.py

```diff
@@ -348,12 +349,32 @@ def __unicode__(self):
         return unicode(self.id)
 
 
+class EmailType(TypeDecorator):
```

Comment:
Maybe let's name it `LowercasedString`?

Reply:
Sure, can rename. Wanted to keep this targeted at email, but if it's more generally useful, then will do.